Small improvements in documentation. (#223)
Microsoft-community/Community-website: src/wiki/backup.md

System Image Backup is another tool you can use to back up your system. A system image is an image, or copy, of the current state of the operating system, stored in a single file. You can use system images to revert the system to a previous state. Note that with System Image Backup, you cannot restore individual files&mdash;you can only roll back the entire system to the state it was in when the image was created.

### Alternative imaging method: Ventoy

> There is an alternative way to back up your system using the free and open source software called Clonezilla for not depending on Windows.

Reviewer: ```suggestion
There is an alternative method to back up your system using the free and open source software Clonezilla, which does not depend on Windows.
```

> [Clonezilla guide of how to create an image of your drive](https://clonezilla.org/fine-print-live-doc.php?path=clonezilla-live/doc/01_Save_disk_image). [And how to restore it](https://clonezilla.org/fine-print-live-doc.php?path=clonezilla-live/doc/02_Restore_disk_image)

Reviewer: This flows awkwardly, "guide of how" is not great English and two hyperlinks back to back with those names is confusing. The one below this works better.

> Before that you should create a USB for booting the live ISO, more details here: ["Creating a Multi-OS Bootable Drive (with Ventoy)"](ventoy.md)

Reviewer: ```suggestion
Before that, you need to create a USB for booting the live ISO. More details here: [Creating a Multi-OS Bootable Drive (with Ventoy)](ventoy.md)
```
